packages/components/src/animate/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+function Animate( { type, options = {}, children } ) {
```

Comment:
Should it exit early when children isn't a function?

Reply:
Same as I was saying above, we don't guard about the other props. I'm hesitant to add a special case for children.